pom.xml

```diff
@@ -65,7 +65,7 @@
     -->
     <tinkerpop.version>3.5.6</tinkerpop.version>
     <!-- when changing version also update version in LICENSE_binary  -->
-    <slf4j.version>1.7.26</slf4j.version>
+    <slf4j.version>2.0.16</slf4j.version>
```

Comment:
Have to bump slf4j version as well here in order to address compile errors re: unable to find org.slf4j.spi.LoggingEvent once we moved to logback 1.3.x.  Per the discussion [here](https://logback.qos.ch/setup.html) it looks like slf4j 2.x is required for logback 1.3.x.